source/_docs/environment-specific-config-d8.md

```diff
@@ -55,6 +55,73 @@ Pantheon handles the inclusion of service configuration files for pre-production
 
 For more information on Pantheon's service configuration files for Drupal, refer to [Creating a services.yml File for Drupal 8](/docs/services-yml).
 
+
+## Enable Cacheability Debugging on Dev & Multidevs
+1. Add the [`sites/default/services.pantheon.preproduction.yml`](https://github.com/pantheon-systems/drops-8/blob/master/sites/default/default.services.pantheon.preproduction.yml) file to your project if you have not done so already.
+
+ This service file is used to manage settings across Pantheon's development environments, like Dev and Multidevs. Settings in this file are not applied to production environments, like Test and Live.
+
+2. Enable Drupal 8's [CacheableResponseInterface](https://www.drupal.org/docs/8/api/responses/cacheableresponseinterface#debugging) to help debug cache by setting the `http.response.debug_cacheability_headers` parameter to `true`:
+
+    ```yaml
+    parameter:
+     http.response.debug_cacheability_headers: true
+    ```
+
+3. Stage, commit, and push your changes to Pantheon:
+
+  ```
+  git commit -am "Pre-production enable cacheability debug service"
+  git push
+  ```
+
+4. Verify service setting and debug cache behavior by inspecting response headers on a development environment URL. If enabled, cacheabile responses will return `X-Drupal-Cache-Tags` and `X-Drupal-Cache-Contexts` headers such as:
+
+  ```bash
+  $ curl -I http://dev-cacheability-headers.pantheonsite.io/
```

Comment:
might make sense to only show relevant headers...maybe start with HTTP/1.1 200 OK, then snip a bunch after that and then show the relevant ones?